content/en/docs/overview/singleton-sdk/Angular Client Introduction/_index.md

```diff
@@ -15,53 +15,222 @@ Besides Localization, singleton client also provides I18n support for Datetime f
 
 #### **Features**
 
-- Internationalization. Such as Dates, Times, Numbers and Currencies formatting support.
-- Localization support.
+- Level-2 Dates, Times, Numbers and Currencies formatting support. For more details refer to i18n Level-2.
+- Level-3 support. For more details refer to i18n Level-3.
 
 
 #### **Version**
 
+- 13.x (support Angular13)
+- 12.x (support Angular12)
+- 11.x (support Angular11)
```

Comment:
remove  below, because currently only support Angular10 on openSource
```
- 13.x (support Angular13)
- 12.x (support Angular12)
- 11.x (support Angular11)
```

Reply:
Done!